azure-pipelines.yml

```diff
@@ -7,6 +7,7 @@ pool:
 variables:
     buildPlatform: 'Any CPU'
     buildConfiguration: 'Release'
+    BuildType: 'Release'
```

Comment:
Nit Use lowercase `b` for consistency with the other two?

Reply:
Sure enough, msbuild doesn't care about the key casing. I originally wanted them to match between `build.proj` and here  but since it works either way, consistency within a file is more important